app/src/main/java/org/piramalswasthya/cho/ui/commons/personal_details/PersonalDetailsFragment.kt

```diff
@@ -122,17 +153,45 @@ class PersonalDetailsFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         HomeViewModel.searchBool.observe(viewLifecycleOwner){
-            bool ->
+                bool ->
             when(bool!!) {
                 true ->{
-                            binding.search.requestFocus()
-                            activity?.window?.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_VISIBLE);
+                    binding.search.requestFocus()
+                    activity?.window?.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_VISIBLE);
 
-                        }
+                }
                 else -> {}
             }
 
         }
+        binding.cameraIcon.setOnClickListener{
+
```

Comment:
we can use generic dialog here that means create dialog in helper file use the same code everywhere so that no need to add seperate code to create dialog in required  file